### PR TITLE
added play.rb file

### PR DESCRIPTION
**Pull request by GASANAMDavid**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Style/Documentation:
```
Comment (GASANAMDavid) [inline .rubocop.yml:1]: Added these configurations in the .rubocop.yml as a result of a rubocop error suggesting that this config info were added to Rubocop 
Screen Shot 2021-05-04 at 17 16 53

File: David_tic_tac_toe.gemspec [hunk 2]
```diff
@@ -12,26 +12,16 @@ Gem::Specification.new do |spec|
...
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.4.0")
-
-  # spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.5.0")
 
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = "https://github.com/GASANAMDavid/David_tic_tac_toe"
   spec.metadata["changelog_uri"] = "https://github.com/GASANAMDavid/David_tic_tac_toe/blob/main/CHANGELOG.md"
 
-  # Specify which files should be added to the gem when it is released.
```
Comment (GASANAMDavid) [inline David_tic_tac_toe.gemspec:23]: Removed unnecessary comments

File: Gemfile [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
```
Comment (GASANAMDavid) [inline Gemfile:3]: change from " " to ' ' robocop suggestion as best practice

File: spec/David_tic_tac_toe_spec.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
-# require_relative "../lib/David_tic_tac_toe"
+
+require_relative "../lib/David_tic_tac_toe"
```
Comment (GASANAMDavid) [inline spec/David_tic_tac_toe_spec.rb:3]: didn't know that this was uncommented

File: lib/play.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+require_relative 'David_tic_tac_toe'
```
Comment (GASANAMDavid) [inline lib/play.rb:1]: Added a new play.rb file utilizing David_tic_tac_toe.rb

File: Gemfile.lock [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 3.0)
+    rubocop-ast (1.5.0)
+      parser (>= 3.0.1.1)
+    ruby-progressbar (1.11.0)
+    unicode-display_width (2.0.0)
+
+PLATFORMS
+  universal-darwin-19
+  x86_64-linux
```
Comment (GASANAMDavid) [inline Gemfile.lock:47]: Adde this x86_64_linux to fix Travis related issue that suggested that my bundle only supports ['universal-darwin-19'] but my local bundle platform is x86_64_linux

**Review by naush — APPROVED** [review]
Looks good to me! Can you research the difference between `require` and `require_relative` and let me know tomorrow?

**Pull request merged**